Library/brew.rb

```diff
@@ -50,6 +50,20 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11" && MacOS::Xcode.version < "7.0"
```

Comment:
I believe the Xcode/CLT doctor checks handle all these cases already. I don't see the benefit of duplicating all of that logic here.